diff --git a/src/builders/protractor/index.js b/src/builders/protractor/index.js
index c3175770fe275393ea2d997c627469c898339754..55a8316756931c22e6d32b471304cb944bebee63 100755
--- a/src/builders/protractor/index.js
+++ b/src/builders/protractor/index.js
@@ -108,17 +108,7 @@ async function execute(options, context) {
             const serverOptions = await context.getTargetOptions(target);
             const overrides = {
                 watch: false,
-                liveReload: false,
             };
-            if (options.host !== undefined) {
-                overrides.host = options.host;
-            }
-            else if (typeof serverOptions.host === 'string') {
-                options.host = serverOptions.host;
-            }
-            else {
-                options.host = overrides.host = 'localhost';
-            }
             if (options.port !== undefined) {
                 overrides.port = options.port;
             }
